ConsistencyManager/ConsistencyManager.swift

```diff
@@ -57,6 +57,11 @@ public class ConsistencyManager {
      Periodically, the consistency manager cleans up after itself.
      This is a relatively quick process, and since the memory footprint of the library in general is small, it's not an important thing to worry about.
      Default is 5 mins. It will also clean up automatically on any memory warnings.
+
+     If you set this to zero, then garbage collection is considered disabled.
+     The ConsistencyManager will still attempt to clean up memory if the application receives a memory warning.
+
+     This variable is not thread-safe. You should only access this on the main thread.
      */
     public var garbageCollectionInterval: NSTimeInterval = 300
```

Comment:
But what would I pass in to sync on? The class doesn't inherit from NSObject. I could create a lock object just to synchronize this, but that seems sort of dirty.

I'd need:
create a lock object
make this a computed property with an private var

^^^ Seems like a lot of effort/extra code to make something faster which doesn't need to be fast?